server/src/controllers/UserChapterRole/resolver.ts

```diff
@@ -0,0 +1,37 @@
+import { Arg, Ctx, Int, Mutation, Resolver } from 'type-graphql';
+import { GQLCtx } from 'src/common-types/gql';
+import { Event, UserChapterRole } from 'src/models';
+
+@Resolver()
+export class UserChapterRoleResolver {
+  @Mutation(() => UserChapterRole)
+  async setUserInterestForChapter(
```

Comment:
@ojeytonwilliams, I named it setUserInterestForChapter , so that in future when we implement the _interested_ toggle functionality, if we want to set interested either to false or true, we could just modify his Mutation to handle that .

Reply:
If we have two functions, `set` and `init`, then someone that just reads the function names should realise that `init` is just used to start things up and `set` will modify something.  That way no one using them should be surprised by their behaviour and I think that's quite valuable.

It will make the code a little less DRY, but if we do it right it shouldn't bloat the file too much.

Does that make sense?

Reply:
Yeah, It make sense @ojeytonwilliams . I'll update the name and merge the changes